utils.py

```diff
@@ -72,3 +74,68 @@ def get_raw_data(fpath_set):
     assert len(raw.annotations) == 1200, err_msg
 
     return raw
+
+
+def get_behavioral_data(fpath_set):
+    """Read behavioral data to DataFrame.
+
+    Does the following:
+        - Reads in behavioral data as csv
+        - Checks if data has the expeceted length
+
+    Parameters
+    ----------
+    fpath_set : string
```

Comment:
`string` is not a datatype, but `str` is:

![image](https://user-images.githubusercontent.com/9084751/163349795-171122d2-269b-43bf-a2f9-51b1a12c0dcd.png)